src/screens/Learn/Faq/Faq.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Wrapper, PageHeader } from './Faq.style';
+import FaqSection from './FaqSection';
+import { faqContent } from 'cms-content/learn';
+
+const faqHeader = faqContent.header;
+const faqSections = faqContent.sections;
+
+const Faq = () => {
+  return (
+    <Wrapper>
```

Comment:
We need to add the page title, canonical URL and description (see the LocationPage for an example)

```tsx
      <AppMetaTags
        canonicalUrl={canonicalUrl}
        pageTitle={pageTitle}
        pageDescription={pageDescription}
      />
```

Reply:
looks like you did this in a separate pr. gonna resolve!